core/src/wallet/ripple/RippleLikeMemo.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ * RippleLikeMemo
+ *
+ * Created by Dimitri Sabadie on 2019/03/26
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Ledger
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#pragma once
+
+#include <string>
+
+namespace ledger {
+    namespace core {
+        // A single memo in a Ripple transaction.
+        struct RippleLikeMemo {
+            std::string data;
+            std::string fmt;
+            std::string ty;
+
+            RippleLikeMemo() = default;
+            RippleLikeMemo(std::string const& data_, std::string const& fmt_, std::string const& ty_);
```

Comment:
Why do you need this constructor? In worse case it can be initialized as {"a", "b", "c"};

Reply:
This file is going to be removed because I had to add this type as an IDL record (so it’ll get generated automatically).